gradle/build-spring.gradle

```diff
@@ -72,5 +72,9 @@ subprojects {
        if (projectType.springDocProjects.contains(project)){
             api(library.springboot_starter_web)
        }
+       
```

Comment:
I am not sure if we need springBootWebUIProjects.
If there is a possiblity to have them ... okay.

But in this case we should add the common parts (e.g. dependencies) here
If there are none, I would prefer to drop those lines.